react/admin/ExportReturnRequestsModal/components/ReportStatus.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { ButtonPlain, IconExternalLink } from 'vtex.styleguide'
+import { FormattedMessage, FormattedDate } from 'react-intl'
+
+import { useExportModule } from '../hooks/useExportModule'
+import TagStatus from './TagStatus'
+
+const ReportStatus = () => {
+  const { exportStatus } = useExportModule()
+
+  const {
+    inProgress,
+    requestedBy,
+    completedDate,
+    selectedFilters,
+    downloadLink,
+    staleLink,
+  } = exportStatus ?? {}
+
+  return (
+    <>
+      <p className="f4 mt2 mb3">
+        <FormattedMessage id="admin/return-app.export-module.report.status-title" />
+      </p>
+      <div className="flex items-center justify-between">
+        <p className="mv3">
+          <ButtonPlain
+            variation="plain"
+            href={downloadLink ?? null}
+            disabled={inProgress || !downloadLink || staleLink}
```

Comment:
The disable key is not making the button unclickable but also the link is valid for more than 6 hours. 🤷‍♂️

Reply:
Yeah... I saw that after I already made everything, I was always working with 'new' reports, it seems that the 6 hour rule ONLY applies to reports made from the OMS and not MASTERDATA. I can go about removing all the values regarding the idea of `staleLink`. But also, reports are suddenly deleted by no apparent reason. It seems that this api is extremely rogue hahaha

Reply:
Regarding the clickable despite disabled, I saw it and forgot to address it. I thought about adding a class to the container like 'pointer-disable' and via CSS set the pointer as none. That fixes it, its a vtex.styleguide bug

Reply:
We can keep the logic of the `staleLink` if we can prevent the user to click it when it is stale. The button is an anchor tag that doesn't have the disable feature on HTML. So it's either via CSS as you mention or removing the href prop when it's stale.

Reply:
Changed the functionality of the `href` prop. If the link is stale, clicking it will do nothing